frontend/javascripts/oxalis/view/action-bar/volume_actions_view.js

```diff
@@ -391,7 +392,13 @@ export default function VolumeActionsView() {
           style={narrowButtonStyle}
           value={VolumeToolEnum.FILL_CELL}
         >
-          <i className="fas fa-fill-drip" />
+          <i
+            className="fas fa-fill-drip"
+            style={{
+              opacity: disabledInfosForTools[VolumeToolEnum.FILL_CELL].isDisabled ? 0.5 : 1,
+            }}
+          />
```

Comment:
Somehow changing the opacity of this fa icon triggers a small transition animation that the icon of the trace tool does not have, as it is not a fa icon.
At first look, I could find a way to avoid the animation. Is it alright to keep the animation?

Reply:
Yes, I think, that's alright :)